buildenv/jenkins/common/variables-functions

```diff
@@ -385,8 +371,6 @@ def set_build_variables() {
     BOOT_JDK = get_value(VARIABLES."${SPEC}".boot_jdk, SDK_VERSION)
     FREEMARKER = VARIABLES."${SPEC}".freemarker
     OPENJDK_REFERENCE_REPO = VARIABLES."${SPEC}".openjdk_reference_repo
-    set_release()
-    set_jdk_folder()
```

Comment:
see comments above (still required - see `stage('Java Version'){...}` in common/build script)